pkg/parser/consts.go

```diff
@@ -30,7 +30,15 @@ const (
 )
 
 const (
-	maxLBRuleCountDefault = 148
-	ipLocalReservedPorts  = 65330
-	MinArgs               = 2
+	maxLBRuleCountDefault               = 148
```

Comment:
these are the sysctl values right?

Reply:
`maxLBRuleCountDefault` is for MAXIMUM_LOADBALANCER_RULE_COUNT and the others `default` are for sysctl